docs/LitActions/helloWorld.md

```diff
@@ -1,11 +1,11 @@
 ---
-sidebar_position: 1
+sidebar_position: 3
```

Comment:
I'd move this up after best practices 
<img width="284" alt="Screenshot 2023-03-21 at 6 35 50 PM" src="https://user-images.githubusercontent.com/5404516/226779364-1a344e10-faee-4918-b396-015e85fb2edc.png">